packages/valory/skills/abstract_round_abci/tests/test_abci_app_chain.py

```diff
@@ -126,6 +132,8 @@ class AbciApp2(AbciApp):
             }
             final_states = {self.round_2c}
             event_to_timeout = {self.event_timeout2: self.timeout2}
+            db_pre_conditions = {self.round_2a: [self.key_1]}  # type: ignore
+            db_post_conditions = {self.round_2c: [self.key_2]}  # type: ignore
```

Comment:
Question is if we should force the dev to make the post conditions strictly include the pre conditions or not. I say we could force that post conditions do **not** include pre conditions as there is no mechanism to delete data from db atm and it is just unnecessary duplication. Also the app does not "generate" this data (it explicitly has it as a pre condition).

Reply:
Yes enforced now.